packages/docs/content/5_components/1_overview.mdx

```diff
@@ -0,0 +1,7 @@
+---
+title: Overview
+---
+
+Component class names follow the format: `ds-c-[BLOCK]__[ELEMENT]--[MODIFIER]`
```

Comment:
Comparing this page to the production one, I realized that the production doc pages use `ds-u-measure--wide`, but these new pages' contents span the full width.

Reply:
I have that noted. The tricky part is that text content is restricted by `ds-u-measure--wide` but examples are not. With the current site, there's not a simple way to add that class to some elements (paragraph, links, tables, etc) but not others (example)

Ideally, the whole page could be restricted by `ds-u-measure--wide`